Pear/Cells/AddProfileSectionTableViewCell.swift

```diff
@@ -44,9 +45,8 @@ class AddProfileSectionTableViewCell: UITableViewCell {
     
     private func setupConstraints() {
         backdropView.snp.makeConstraints { make in
-            make.verticalEdges.equalToSuperview().inset(4)
+            make.leading.trailing.equalToSuperview()
```

Comment:
Try to keep changes unrelated to the focus of this PR in a separate PR. Appreciate the attention to detail though.

Reply:
Oops this PR was supposed to be stacked on top of the other one, thanks for pointing that out.